app/controllers/api/educations_controller.rb

```diff
@@ -1,2 +1,45 @@
 class Api::EducationsController < ApplicationController
+  def index
+    @educations = Education.all
```

Comment:
Education all retrieves every example of education in the database. It should be changed to current_student.id.